src/ui/components/NetworkMonitor/RequestTable.module.css

```diff
@@ -4,12 +4,21 @@ table.requests {
 
 .row {
   position: relative;
+  border: 2px solid transparent;
+}
+
+.row:hover {
+  cursor: pointer;
 }
 
 .row:not(:first-child) {
   border-top: 2px solid var(--theme-toolbar-background);
 }
 
+.row.selected {
+  border: 2px solid var(--primary-accent);
```

Comment:
we should use the same row selected treatment as other places: blue background, white text which is also what chrome does too.